### ISSUE TITLE
Could you create the sidebar at the same position like the drop target?

### ISSUE THREAD

**Issue by ssborbis** [ssborbis/ContextSearch-web-ext#109]
Because I cannot leave it open at the top right corner, because there are always buttons.
Not sure how to do this at the bottom, probably bottom of sidebar = bottom of drop target.
Dragging the sidebar would also be nice...

_Originally posted by @hg42 in https://github.com/ssborbis/ContextSearch-web-ext/issues/104#issuecomment-439656189_

**Comment by ssborbis** [comment]
The code gets a bit complicated when accounting for positions relative to the top OR bottom. Originally I toyed with checking for drop target position > 50% window height to use style.top, < 50% use style.bottom but I have to account for sidbar menu height change when going into subfolders, transform-origin for scaling, etc. I scrapped it for something simpler that worked I could put into repo for testing. I'll give it another look though.

> Dragging the sidebar would also be nice...

I guess the menu should have a separate offset from the widget. Unfortunate but probably unavoidable
